src/user/user.interface.ts

```diff
@@ -1,6 +1,12 @@
 export interface IUser {
 	id: string;
```

Comment:
This interface contains first name and last name twice. Once inside of an object and then outside, it should be an optional.